apps/publish/formatters/aap_ipnews_formatter.py

```diff
@@ -59,7 +59,7 @@ def format(self, article, subscriber):
                 else:
                     odbc_item['subject_reference'] = '00000000'
 
-                odbc_item['take_key'] = article.get('anpa_take_key', None)  # @take_key
+                odbc_item['take_key'] = article.get('anpa_take_key', None).replace('\'', '\'\'')  # @take_key
```

Comment:
it should be  `odbc_item['take_key'] = (article.get('anpa_take_key', '') or '').replace('\'', '\'\'')`